src/account.ts

```diff
@@ -676,45 +675,45 @@ export default class EthereumAccount {
 
           const claim = JSON.parse(machinomy.data.toString())
 
-          const hasValidSchema = claim
-            && typeof claim.value === 'string'
-            && typeof claim.channelId === 'string'
-            && typeof claim.signature === 'string'
-          if (!hasValidSchema) {
+          const hasValidSchema = (o: any): o is IClaim => {
```

Comment:
Why does this need to be a closure instead of just an if statement?

Reply:
I wasn't able to find a way to get the type guard `o is IClaim` to work correctly unless it was a function -- any ideas? (Otherwise `claim` will just be `any`).